content/testing-yaml/firebase-test-lab.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Testing on Firebase Test Lab
+description: How to run tests on Firebase Test Lab
+weight: 2
+---
+
+Firebase Test Lab provides cloud-based infrastructure for testing Android and iOS apps on a range of devices and configurations. You can test your apps on Firebase Test Lab as part of the Codemagic build workflow provided you have set up a Firebase project.
+
+To set up testing on Firebase Test Lab in Codemagic, follow these steps.
+
+## Firebase Test Lab prerequisites
+
+You will need to set up the following in the [Firebase console](https://firebase.google.com/).
+
+1. Create a Firebase project.
+2. Create a service account with Editor permissions and create a key. Download the JSON key file so you can authenticate with Firebase Test Lab during the build.
```

Comment:
Do we have a link to any instructions/existing docs on how to do this?

Reply:
Nothing besides the blog post as far as I know.

Reply:
Turns out this doc actually describes creating the service account, I've linked to it: https://docs.codemagic.io/knowledge-base/google-play-api/. Will make the file more generic in a separate PR.